openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -980,13 +980,13 @@ For the errors specific to the payload of the Credential Request such as those c
 
 If the Wallet is requesting the issuance of a Credential that is not supported by the Credential Endpoint, the HTTP response MUST use the HTTP status code 400 (Bad Request) and set the content type to `application/json` with the following parameters in the JSON-encoded response body:
 
-* `error`: REQUIRED. A key at the top level of the object, the value of which SHOULD be a single ASCII [@!USASCII] error code from the following:
+* `error`: REQUIRED. A name at the top level of the object, the value of which SHOULD be a single ASCII [@!USASCII] error code from the following:
```

Comment:
Don't think "name" fits well here..

Reply:
All items in the list are parameters and are at the top level. We can simplify this as follows:
```suggestion
* `error`: REQUIRED. The `error` parameter SHOULD be a single ASCII [@!USASCII] error code from the following:
```

Reply:
+1 to @awoie's suggestion here